sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/export/_base.py

```diff
@@ -162,6 +173,24 @@ def __init__(self, **kwargs: Any) -> None:
 
             collect_statsbeat_metrics(self)
 
+        # Initialize customer statsbeat if enabled
+        self._customer_statsbeat_metrics = None
+        if self._should_collect_customer_statsbeat():
+            try:
+                statsbeat_options = type('StatsbeatOptions', (), {
+                    'instrumentation_key': self._instrumentation_key,
+                    'endpoint_url': self._endpoint,
+                    'network_collection_interval': kwargs.get('statsbeat_interval', 900000)
```

Comment:
Statsbeat interval isn't configurable at least for now and should always be 15m.